tutos/dev/manual/application.wiki

```diff
@@ -833,29 +856,40 @@ CLIENT_PACKAGES := ... ocsigen-widgets.client
 >>
 
 To create the widget, we add the following code in the
-{{{init_client}}} immediately after canvas configuration:
+{{{page}}} :
 
 <<code language="ocaml" class="client"|
 (* Color of the brush *)
-let colorpicker = Ow_color_picker.create ~width:150 () in
-Ow_color_picker.append_at (Dom_html.document##.body) colorpicker;
-Ow_color_picker.init_handler colorpicker;
+  let colorpicker, cp_sig =
+    Ot_color_picker.make ~a:[Html.D.a_class ["colorpicker"]]
 >>
 
 We subsequently add a simple HTML slider to change the size of the
 brush. Near the {{{canvas_elt}}} definition, simply add the following
 code:
 
 <<code language="ocaml" class="server"|
-let slider =
-  Eliom_content.Html.D.Form.input
-    ~a:[
-      Html.D.a_id "slider";
-      Html.D.a_input_min (`Number 1);
-      Html.D.a_input_max (`Number 80)
-    ]
-    ~input_type:`Range
-    Html.D.Form.int
+let%server slider_create () =
+  let slider_sig, slider_f = Eliom_shared.React.S.create 0 in
+  let elt =
+    let open Html.F in
+    input
+      ~a:
+        [ a_input_type `Range
+        ; a_class ["slider"]
+        ; a_input_min (`Number 1)
+        ; a_input_max (`Number 80)
+        ; a_value "1"
+        ; a_oninput
+            [%client
```

Comment:
If I'm right, we never explained this syntax before.
Could you use the same mechanism as before to get the slider value?